app/scripts/models/notifications.js

```diff
@@ -15,7 +15,9 @@ define([
   var EVENTS = {
     DELETE: 'fxaccounts:delete',
     LOGOUT: 'fxaccounts:logout',
-    PROFILE_CHANGE: 'profile:change'
+    PROFILE_CHANGE: 'profile:change',
+    SIGNED_IN: 'fxaccounts:signedIn',
+    SIGNED_OUT: 'fxaccounts:signedOut'
```

Comment:
@philbooth - I think we should be good, `LOGOUT` is invoked by [`loggedOut`](https://github.com/mozilla/fxa-content-server/pull/3232/files#diff-f997f849acbac327a997c849e2c5b027R54) which is never invoked. `fxaccounts:logout` is listened for [by FxDesktop on the WebChannel](https://dxr.mozilla.org/mozilla-central/source/services/fxaccounts/FxAccountsWebChannel.jsm#29), though for accounts used to sign into Sync, we do not yet surface the ability for the user to sign out. With multi device, we will. Maybe the thing to do here is to use the alias `SIGNED_OUT` and the message `fxaccounts:logout`.